apps/docs/src/app/platform/component-docs/platform-forms/platform-date-picker/platform-date-picker-examples/platform-date-picker-complexi18n.component.ts

```diff
@@ -0,0 +1,170 @@
+/**
+ * This approach has been taken from core/datepicker implementation.
+ * Some part of code has been modified to integrate platform capabilities.
+ */
+
+import { Component, Injectable, ViewChild } from '@angular/core';
+import { CalendarI18n, FdDate } from '@fundamental-ngx/core';
+import { DatePickerComponent } from '@fundamental-ngx/platform';
+
+import * as moment from 'moment';
+import 'moment/locale/es';
+import 'moment/locale/en-gb';
+import 'moment/locale/de';
+import 'moment/locale/fr';
+import 'moment/locale/bg';
+import 'moment/locale/pl';
+import { registerLocaleData } from '@angular/common';
+import localeFrench from '@angular/common/locales/fr';
+import localePolish from '@angular/common/locales/pl';
+import localeBulgarian from '@angular/common/locales/bg';
+import localeGb from '@angular/common/locales/en-GB';
+import localeDe from '@angular/common/locales/de';
+
+@Injectable()
+export class CustomI18nMomentCalendar extends CalendarI18n {
+    getDayAriaLabel(date: Date): string {
+        return date.getDate() + ' ' + moment.months()[date.getMonth()] + ' ' + date.getFullYear();
```

Comment:
this `aria label` is not formatted in a localized way. For other locales we are only putting the localization in the visible aspects of the input?

Reply:
Made changes. using moment(date).format('L') now.